src/words/environment/WordsEventListener.java

```diff
@@ -18,29 +19,39 @@ public WordsEventListener(AST predicate, AST statementList, boolean temporary) {
 	}
 	
 	public boolean execute(WordsEnvironment environment) throws WordsProgramException {
```

Comment:
For clarity we should have Javadoc documentation here indicating that it returns true if it should be deleted and false otherwise.